app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+from datetime import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Should be vaccinated")
+        else:

```

Comment:
I think it looks better if you rewrite this part without else
Avoid using unnecessary `else`